cmake/traccc-acts.cmake

```diff
@@ -0,0 +1,31 @@
+# TRACCC library, part of the ACTS project (R&D line)
+#
+# (c) 2021 CERN for the benefit of the ACTS project
+#
+# Mozilla Public License Version 2.0
+
+# Guard against multiple includes.
+include_guard(GLOBAL)
+
+# Try to find ACTS installed on the system.
+find_package(Acts QUIET)
+
+# If it was found, then we're finished.
+if(Acts_FOUND)
+   # Call find_package again, just to nicely print where it is picked up from.
+   find_package(Acts)
+   return()
+endif()
+
+# Tell the user what's happening.
+message(STATUS "Building ACTS as part of the traccc project")
```

Comment:
:smile: I think the Acts team would object to calling Acts "ACTS". :rofl:

Reply:
Wait, is "ACTS" not the proper capitalization? Now I'm confused! The paper (https://arxiv.org/abs/2106.13593) does capitalize it that way! :sob:

Reply:
Ugh. This has been a bit of back and forth. We've converged on referring to it as *Acts* in the code since we follow (I believe it's) Google's guideline to not write all-caps acronyms when part of names. When referring to the project, we decided to use *ACTS* in the paper.

Reply:
Alright, I'll change it to read "Acts" then! :smile: